packages/protocol/contracts/governance/Validators.sol

```diff
@@ -258,21 +266,35 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
       bytes(identifier).length > 0 &&
       bytes(name).length > 0 &&
       bytes(url).length > 0 &&
-      publicKey.length == 64
+      // secp256k1 public key + BLS public key + BLS proof of possession
+      publicKeysData.length == (64 + 48 + 96)
     );
+    bytes memory proofOfPossessionBytes = publicKeysData.slice(64, 48 + 96);
+    // Disabled until we have a TypeScript version
```

Comment:
Just to follow conventions, I'd recommend removing commented out code and instead leaving a `// TODO(Kobi): ...` comment.